### PR TITLE
Move verb to request object

### PR DESCRIPTION
**Pull request by eduardostalinho**
This PR moves `verb` references to `Request` initialization.
The name `method` does not make sense in this context anymore, all references to this name were changed to `action`.

### CODE REVIEW

File: pactum/request.py [hunk 1]
```diff
@@ -1,5 +1,9 @@
 class Request:
-    def __init__(self, payload=None, headers=None):
+    def __init__(self, verb=None, payload=None, headers=None):
+        if verb is None:
+            verb = getattr(self, 'verb', '')
```
Comment (daneoshiga) [inline pactum/request.py]: é ok um request com verb `''` ?

Comment (osantana) [reply pactum/request.py]: acho que não :/

File: pactum/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from .action import Action  # NOQA
+from .api import API  # NOQA
+from .resources import ListResource, Resource  # NOQA
+from .response import Response  # NOQA
+from .request import Request  # NOQA
+from .route import Route  # NOQA
+from .version import Version  # NOQA
```
Comment (osantana) [inline pactum/__init__.py]: seria melhor colocar essa mudança em um PR separado...

File: tests/test_request.py [hunk 3]
```diff
@@ -1,19 +1,23 @@
 from pactum.request import Request
+from pactum.verbs import GET
 
 
 def test_basic_request():
-    request = Request(payload=None, headers=[])
+    request = Request(verb=GET, payload=None, headers=[])
 
+    assert request.verb == GET
```
Comment (osantana) [inline tests/test_request.py]:
```python
    assert request.verb == verbs.GET
```

File: tests/test_route.py [hunk 4]
```diff
@@ -4,37 +4,37 @@
...
 def test_prefer_parameter_to_class_definition():
     class TestRoute(Route):
         path = '/test/'
-        methods = []
+        actions = []
 
     route = TestRoute(
         path="/test_by_param/",
-        methods=['GET']
+        actions=['GET']
```
Comment (osantana) [inline tests/test_route.py]: Is it correct? Maybe:

```python
        actions=[verbs.GET],
```

Comment (eduardostalinho) [reply tests/test_route.py]: The correct way is
```python
    actions=[Action(verb=GET),]
```

Comment (osantana) [reply tests/test_route.py]: hummm... yes... you're right...

Comment (eduardostalinho) [reply tests/test_route.py]: Not event this is correct.

File: pactum/request.py [hunk 5]
```diff
@@ -1,5 +1,12 @@
 class Request:
-    def __init__(self, payload=None, headers=None):
+    def __init__(self, verb=None, payload=None, headers=None):
+        if verb is None:
+            try:
+                verb = getattr(self, 'verb')
+            except AttributeError:
+                raise TypeError('Missing request verb.')
```
Comment (osantana) [inline pactum/request.py]:
```python
                raise TypeError('Missing request verb specification.')
```

**Pull request merged**